publishedbranches/docs-atlas-operator.yaml

```diff
@@ -0,0 +1,15 @@
+prefix: 'atlas-operator'
```

Comment:
question: when we add a new branch in Hapley will this be updated?

Reply:
No, that process is not automated actually. Each of these are added manually for now.

Reply:
@mmeigs This file isn't used/needed anymore (we source this info from Hapley), so we can skip it entirely.

Reply:
Nevermind! See below!